_episodes/05-act-actel.md

```diff
@@ -504,293 +502,4 @@ actel_explore_output_chesapeake <- explore(datapack=actel_project, report=TRUE,
 n
 ~~~
 
-If you haven't been able to follow along, this report is included in the `data/act/` folder you have cloned from the workshop repository (called actel_explore_report_chesapeake.html) so you can check it out!
```

Comment:
keep this line for the students, i added the report just in case they wanna check it out! its specific for the ACT group

Reply:
jk again, i see its at the bottom of the diff ahaha